### PR TITLE
Conditions

### PR DESCRIPTION
**Pull request by lxxxvi**
I think this one's ready to be reviewed. 

I've taken over everything from the existing slides with a new, consistent example though.
Also added a "Common Mistakes" section, but I am not really sure about this part... what do you think?

### CODE REVIEW

File: slides/conditions/index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  puts "No rain, no sun"
+end
+</code></pre>
+          </section>
+        </section>
+
+        <section>
+          <h2>If-Then-Else Summary</h2>
+          <div>
+<pre class="fragment fade-in"><code class="hljs ruby">if (condition1)
```
Comment (nerdinand) [inline slides/conditions/index.html]: Maybe leave the parentheses off, for consistency?

File: slides/conditions/index.html [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  # this part is executed if condition1 was true
+elsif (condition2)
```
Comment (nerdinand) [inline slides/conditions/index.html]: dito

File: slides/conditions/index.html [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+        </section>
+
+        <section>
+          <section>
+            <h2>Common mistakes</h2>
+            <ul>
+              <li class="fragment fade-in">Confusion btw assignment (<b>=</b>) and comparison (<b>==</b>)</li>
+              <li class="fragment fade-in">Not providing the <b>end</b> key word</li>
+              <li class="fragment fade-in"><b>elsif</b> (not <b>elseif</b> or <b>else if</b>)</li>
+              <li class="fragment fade-in">Not using indendation for visual orientation</li>
```
Comment (nerdinand) [inline slides/conditions/index.html]: typo: indendation

File: slides/conditions/index.html [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+          </div>
+          <aside class="notes">
+          </aside>
+        </section>
+
+        <section>
+          <h2>Conditions</h2>
+          <div class="flex-container">
+            <div class="flex-column">
+              <p>We use conditions to <b class="text-blue">change the behaviour</b> depending on one or multiple <b>input</b></p>
```
Comment (nerdinand) [inline slides/conditions/index.html]: typo: input --> inputs

**Comment by lxxxvi** [comment]
@nerdinand 

I don't think we should mention truthy/falsey here already... I think at this stage it's more confusing than helpful. I started adding a new slide (see below) and it felt wrong. Also I think (personal opinion) we should encourage students using real booleans whenever possible.

Are there other opinions?

--- 

```html
        <section>
          <h2>truthy or falsey</h2>
          <p>Ruby does not necessarily need a "proper boolean" for <b>if-then-else</b> to work.</p>
          <p class="fragment fade-in">Ruby uses a concept of <b>truthy</b> and <b>falsey</b></p>
          <p class="fragment fade-in">The strategy is pretty simple: Everything is <b>truthy</b>, except <b>nil</b> and <b>false</b></p>

<pre class="fragment fade-in"><code class="hljs ruby">if "banana"                 # this is truthy
  puts "I like bananas!"    # this is executed for sure
end

if nil                      # this is falsey
  puts "I like the void!"   # this won't be executed
end
</code></pre>
        </section>
```

**Comment by lxxxvi** [comment]
@nerdinand however... I added a new slide about `true` and `false`, as this was not properly explained yet.

**Comment by nerdinand** [comment]
@lxxxvi good idea, i like the additional true/false slide. but i'd make it an interactive quiz and only reveal the solution after an additional click

**Comment by lxxxvi** [comment]
@hharen @nerdinand Thanks for your reviews 🐙 I corrected your mentioned things and made the quiz. It doesn't look super-beautifully, but it should be good enough.

File: slides/conditions/index.html [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+            <li class="fragment fade-in">If <b>you're hungry</b>, <b class="text-blue">eat an apple</b></li>
+            <li class="fragment fade-in">If <b>the payment was successful</b>, <b class="text-blue">send a mail</b></li>
+          </ul>
+
+          <aside class="notes"></aside>
+        </section>
+
+        <section>
+          <h2>true or false</h2>
+          <p>Before we can use <b>if</b>, we need a to have something that is either <b>true</b> or <b>false</b></p>
```
Comment (nerdinand) [inline slides/conditions/index.html]:
```suggestion
          <p>Before we can use <b>if</b>, we need to have something that is either <b>true</b> or <b>false</b></p>
```

**Pull request merged**